### PR TITLE
feature(Notifications): Add notification settings card component

### PR DESCRIPTION
**Pull request by kunstefix**
## Checklist

- [ ] I have read the **README** document
- [ ] I have read the **CONTRIBUTING** document
- [ ] I have added tests to cover my changes
- [ ] All new and existing tests passed
- [ ] My commit message follows [Conventional Commit Guideline](https://conventionalcommits.org/)

### CODE REVIEW

File: libs/design-system-components/src/components/NotificationSettingsCard/index.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    <Card radius={16} customStyle="p-0 w-full grow flex-wrap">
+      <Stack padding="p-5" align="center">
+        {imageSrc ? (
+          <Image
+            customStyle="w-[180px] h-[180px] object-contain mb-4"
+            src={imageSrc}
+            alt="Notification illustration"
+          />
+        ) : (
+          <div aria-hidden="true" className="w-[180px] h-[180px] bg-[#C4C4C4] mb-4"></div>
```
Comment (didd) [inline libs/design-system-components/src/components/NotificationSettingsCard/index.tsx:38]: We have predefined colors https://github.com/AKASHAorg/akasha-core/blob/next/libs/typings/src/ui/design-system.ts. Is #C4C4C4 what you saw on the design system for this card ?

Comment (kunstefix) [reply libs/design-system-components/src/components/NotificationSettingsCard/index.tsx:38]: Yes, #C4C4C4 is in the Figma, but i didn't see variable defined anywhere. Should I  define a new one or do we allow such excemptions?

Comment (didd) [reply libs/design-system-components/src/components/NotificationSettingsCard/index.tsx:38]: I see. If its a new variation of grey can you define it in the variables ?

Comment (didd) [reply libs/design-system-components/src/components/NotificationSettingsCard/index.tsx:38]: Ok I asked Basma, I now understand the situation, so this div is a placeholder till she finds a proper image on the design it was not meant to be created as you did. For such cases you can just leave a comment and when she gets the proper image you can replace it with a relevant component.

**Review by didd — DISMISSED** [review]

**Review by quininez — DISMISSED** [review]

**Review by didd — APPROVED** [review]

**Review by quininez — APPROVED** [review]

**Pull request merged**